client/react/app/reducers/index.js

```diff
@@ -1,9 +1,152 @@
-// TODO: implement me!
-const reducer = (state = {}, action = {}) => {
+import { combineReducers } from 'redux';
+import { extend } from 'lodash';
+import { fetchScorecards } from '../helpers/scorecard';
+import { fetchChecklists } from '../helpers/checklist';
+import { mapByDate } from '../helpers/utils';
+
+// Constants
+
+export const LIST_VIEW = 'LIST_VIEW';
+export const CHART_VIEW = 'CHART_VIEW';
+
+export const UPDATE_VIEW = 'UPDATE_VIEW';
+export const SELECT_DATE = 'SELECT_DATE';
+export const REQUEST_SCORECARDS = 'REQUEST_SCORECARDS';
+export const RECEIVE_SCORECARDS = 'RECEIVE_SCORECARDS';
+export const REQUEST_CHECKLISTS = 'REQUEST_CHECKLISTS';
+export const RECEIVE_CHECKLISTS = 'RECEIVE_CHECKLISTS';
+
+// Actions
```

Comment:
(eventually it might be better to split these out into separate files, but in the meantime I think this is fine)